_includes/_code/tapbox-d/main.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+// State is managed by TapboxModel, a subclass of ChangeNotifier.
+// The StatefulWidgets register as listeners.
+// TBD: Adam is going to add ListeningWidget to the framework,
+//      so it won't be needed here.
+// If something changes, TapboxModel notifies its listeners.
+
+//---------------------------- ParentWidget ----------------------------
+
+class ParentWidget extends StatefulWidget {
+  @override
+  ParentWidgetState createState() => new ParentWidgetState();
+}
+
+class ParentWidgetState extends State<ParentWidget> {
+  TapboxModel model = new TapboxModel();
+
+  @override
+  Widget build(BuildContext context) {
+    return new Column(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        new TapboxD(model: model),
+        new TapboxD(model: model),
+      ],
+    );
+  }
+}
+
+//-------------------------- ListeningWidget ---------------------------
```

Comment:
You can remove `ListeningWidget` now.  I generalized `AnimatedWidget` in the framework to do this work.